.gitlab-ci.yml

```diff
@@ -47,6 +47,27 @@ lint:
   script:
     - make verify
 
+nocuda-tests:
+  stage: test
+  image: $DOCKER_TAG
+  needs: ["lint"]
+  tags:
+    - timemachine
+    - cpu
+  rules:
+    - if: $CI_EXTERNAL_PULL_REQUEST_IID
+  variables:
+    CMAKE_ARGS: -DCUDA_ARCH=75
```

Comment:
Don't need the Cuda arch

Reply:
Removed in https://github.com/proteneer/timemachine/pull/1108/commits/3e45db9e90882bddfc0a7b3271d1e0728879d4cd.